scripts/synthetic/generate_pipeline_results_files.py

```diff
@@ -160,53 +163,66 @@ def _generate_resultfiles_json(
     help="the sample metadata file",
 )
 @click.option(
-    "--output-csv-file",
+    "--output-results-csv-file",
     type=click.Path(dir_okay=False, writable=True),
-    required=True,
-    help="output file merging the results from ncov and pangolin",
+    default="results.csv",
+    help="CSV file containing the summary of results",
 )
 @click.option(
-    "--output-json-file",
+    "--output-results-json-file",
     type=click.Path(dir_okay=False, writable=True),
-    required=True,
-    help="output file containing all the expected files per sample",
+    default="results.json",
+    help="JSON file containing the summary of results",
+)
+@click.option(
+    "--output-resultfiles-json-file",
+    type=click.Path(dir_okay=False, writable=True),
+    default="resultfiles.json",
+    help="JSON file containing all the expected files per sample",
 )
 @click.option(
     "--output-path",
     type=str,
     required=True,
     help="output_path output path where sample result files are stored (e.g. s3://bucket/path/analysis_run)",
 )
-def generate_results(
+def generate_pipeline_results_files(
     metadata_file: str,
-    output_csv_file: str,
-    output_json_file: str,
+    output_results_csv_file: str,
+    output_results_json_file: str,
+    output_resultfiles_json_file: str,
```

Comment:
The pipeline generates 3 files: 
- results.csv 
- results.json ( this is like results.csv but in json format as there was rumours to switch to json last November).
- resultfiles.json

Reply:
I've updated the description for `--output-results-json-file`, but prefer to keep the variable names as they are because they reflect the actual file names.